infoblox/infoblox.py

```diff
@@ -1306,7 +1313,10 @@ def get_a_record_by_fqdn(self, fqdn):
     def update_record(self, record, fields, confirm):
         self.util.put(record, fields, confirm)
 
-    def get_range(self, network, fields=None, not_found_fail=True):
```

Comment:
Just for backwards compatibility, it would be good to make `get_range` an alias to `get_dhcp_range`.

Reply:
This was added by me in my last PR -- it's never been released so it shouldn't be a backwards compatible issue:
https://github.com/EFXCIA/Infoblox-API-Python/commit/edf9a28b19e306295cea9f6adea795e34b5040fb

Reply:
Ah, cool.